build.gradle

```diff
@@ -107,23 +107,98 @@ allprojects {
     }
 
     tasks.withType(JavaCompile) {
+        options.compilerArgs += ['-Werror']
         // temporarily relax constraints until we can fix all violations
-        options.errorprone.warn 'CatchBlockLogException',
+        options.errorprone.disable 'AlmostJavadoc', 
+                'AnnotateFormatMethod',
+                'ArrayAsKeyOfSetOrMap',
+                'AssertEqualsArgumentOrderChecker',
+                'AssertionFailureIgnored',
+                'ByteBufferBackingArray',
```

Comment:
Potential data correctness issues, we should fix the underlying code and enforce this asap.